utils/sql-db-seeder/src/index.ts

```diff
@@ -3,51 +3,73 @@ import { SqlDbDataSource } from '@ukef/dtfs2-common/sql-db-connection';
 import { AzureFileInfoEntity, FacilityUtilisationDataEntity, FeeRecordEntity, PaymentEntity, UtilisationReportEntity } from '@ukef/dtfs2-common';
 import { seedUtilisationReports } from './utilisation-report';
 import { seedFeeRecordPaymentGroups } from './fee-record-payment-group';
-import { mongoDbClient } from './mongo-db-client';
+import { mongoDbClient } from '../../drivers/db-client';
 
 /**
- * Clears the database
- * @param dataSource - The SQL data source
+ * Clears all data from the specified MSSQL database.
+ *
+ * @param {DataSource} dataSource - The data source representing the MSSQL database.
+ * @returns {Promise<void>} - A promise that resolves when the database has been cleared.
  */
 const clearDatabase = async (dataSource: DataSource): Promise<void> => {
+  console.info('⚡ Clearing MSSQL database');
+
   await dataSource.manager.delete(PaymentEntity, {});
   await dataSource.manager.delete(FeeRecordEntity, {});
   await dataSource.manager.delete(UtilisationReportEntity, {});
   await dataSource.manager.delete(AzureFileInfoEntity, {});
   await dataSource.manager.delete(FacilityUtilisationDataEntity, {});
+
+  console.info('✅ MSSQL database has been cleared successfully');
 };
 
 /**
- * Seeds data
- * @param dataSource - The SQL data source
+ * Seeds the specified MSSQL database with initial data.
+ *
+ * @param {DataSource} dataSource - The data source representing the MSSQL database.
+ * @returns {Promise<void>} - A promise that resolves when the database has been seeded.
  */
 const seedData = async (dataSource: DataSource): Promise<void> => {
+  console.info('⚡ Seeding MSSQL database');
+
   await seedUtilisationReports(dataSource);
   await seedFeeRecordPaymentGroups(dataSource);
+
+  console.info('✅ MSSQL database has been seeded successfully');
 };
 
 /**
- * Runs the seeder
+ * Runs the database seeding process, including clearing the database and seeding it with initial data.
+ *
+ * @returns {Promise<void>} - A promise that resolves when the seeding process is complete.
  */
 const run = async (): Promise<void> => {
-  const dataSource = await SqlDbDataSource.initialize();
+  const dataSource = (await SqlDbDataSource.initialize()) as unknown as DataSource;
```

Comment:
Was there no way to fix this without using `as`?
If we need to use `as` can we just do `as DataSource` and skip the casting to unknown before we cast to DataSource as that's surprising and shouldn't be needed I would have thought

Reply:
Apologies, this should have not been committed, it was a local test.
Have set the static type for `dataSource` variable to TypeORM `DataSource`.

Reply:
Nice!